lib/puppet/feature/aws.rb

```diff
@@ -0,0 +1,3 @@
+require 'puppet/util/feature'
+
+Puppet.features.add(:aws, libs: 'aws-sdk-core')
```

Comment:
This will likely break anything pre Ruby-2.0, please use syntax found in the other feature files: `:libs => ['aws-sdk-core']`